paging/common/src/test/kotlin/androidx/paging/InvalidatingPagingSourceFactoryTest.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.paging
+
+import org.junit.Test
+import kotlin.test.assertEquals
+
+class InvalidatingPagingSourceFactoryTest {
+
+    @Test
+    fun getPagingSource() {
+
+        val testFactory = getFactory()
+
+        for (i in 0..3) {
```

Comment:
Bit more idiomatic is `repeat`: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/repeat.html

Sorry should have mentioned in the first pass.